src/Workspaces/Core/Portable/Utilities/SerializableBytes.cs

```diff
@@ -238,7 +238,7 @@ public byte[] ToArray()
             public ImmutableArray<byte> ToImmutableArray()
             {
                 var array = ToArray();
-                return ImmutableArrayExtensions.DangerousCreateFromUnderlyingArray(ref array);
+                return Roslyn.Utilities.ImmutableArrayExtensions.DangerousCreateFromUnderlyingArray(ref array);
```

Comment:
It looks like this is the only remaining use. Implementing this directly would be only a few lines of code -- want to either just update it or file a bug tracking the cleanup?

Reply:
I have it recorded that Tomas wants this function available in the Workspaces layer, so I'll keep it around. He can remove it if he wants